src/fortran_routing/mc_pylink_v00/MC_singleSeg_singleTS/MCsingleSegStime_f2py_NOLOOP.f90

```diff
@@ -173,7 +173,8 @@ subroutine secant_h0(z, bw, bfd, twcc, s0, n, ncc, dt, dx, &
     real(prec), intent(in) :: dt, dx
     real(prec), intent(in) :: qdp, ql, qup, quc
     real(prec), intent(in) :: h_0  !, refQj_0
-    real(prec), intent(out) :: WPC, Qj_0, C1, C2, C3, C4
+    real(prec), intent(out) :: Qj_0, C1, C2, C3, C4
+    real(prec), intent(inout) :: WPC
```

Comment:
I didn't realize these were still in here. I am trying to figure out why Juzer feels so strongly about the WPC initialization. Let me check with him again.

Reply:
This detail has no effect on how we wrap the fortran code.